content/integrations/ntp.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Datadog-NTP Check Integration
+integration_title: NTP Check
+kind: integration
+doclevel: basic
+newhlevel: true
+---
+
+# Overview
+
+The Network Time Protocol (NTP) integration is enabled by default and reports the time offset from an ntp server every 15 minutes.
```

Comment:
There some good info in this KB article on why correctly synced clocks are important to properly functioning Datadog.  Lets either include some of this or link to it: https://help.datadoghq.com/hc/en-us/articles/204282095-Network-Time-Protocol-NTP-Offset-Issues